website/docs/Publishing.md

```diff
@@ -13,6 +13,21 @@ If you are using git and Github pages, you can publish your doc website with one
 
 Otherwise, you can run `moonwave build`, and your built website will be in a folder called `build` in your project directory.
 
+## Using GitHub Actions
+
+When you publish your doc via GitHub Actions you might need to provide the author details and GitHub token.
+
+```yaml
+- name: Publish
```

Comment:
I would include the full example of a Moonwave GitHub Action, and move it down on the page since it has low importance. It's nice to include a working example of things to prevent confusion, we can also specify where they should put this in their repository in the description for this section.